### ISSUE TITLE
book: change `println` to `fmt.Println`

### ISSUE THREAD

**Issue by Delta456** [go101/go101#155]
Hey! Thanks for writing such an awesome book but I had noticed that you have used `print` and `println` for printing to `stdout`. These function use should be discouraged as its for bootstrapping and it can be removed whenever Go Team wants to.

I prefer we should switch to `fmt.Println()` and `fmt.Print()`.

**Comment by go101** [comment]
Yes, the two builtin functions are only used in the starting chapters (before the "Code Packages and Package Imports" chapter) to a void explaining packages and importing in those chapters.

> can be given in `Some Special Topic`.

Do you mean move this note down somewhere in the "Introduction to Source Code Elements" chapter?

**Comment by Delta456** [comment]
Yes

**Comment by go101** [comment]
Maybe, showing the note in a special style would be better.
There are many notes in Go 101 articles.
They are deserved a style which could attract more attention.

**Comment by Delta456** [comment]
Yeah
